projects/plugins/jetpack/_inc/client/components/admin-notices/style.scss

```diff
@@ -42,4 +42,13 @@
 	.dops-notice__action.notice-dismiss {
 		height: 100%;
 	}
+
+	.jp-heading-sr-only {
```

Comment:
Do we need to add a new class for this or can we use `.screen-reader-text`?

https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/